CONTRIBUTING.md

```diff
@@ -1,36 +1,36 @@
-# Contributor's Guide
+# Contributor's guide
 
 We welcome pull requests from cmd2 users and seasoned Python developers alike! Follow these steps to contribute:
 
-1. Find an issue that needs assistance by searching for the [Help Wanted](https://github.com/python-cmd2/cmd2/labels/help%20wanted) tag.
+1. Find an issue that needs assistance by searching for the [Help Wanted](https://github.com/python-cmd2/cmd2/labels/help%20wanted) tag
 
-2. Let us know you are working on it by posting a comment on the issue.
+2. Let us know you're working on it by posting a comment on the issue
 
-3. Follow the [Contribution Guidelines](#contribution-guidelines) to start working on the issue.
+3. Follow the [Contribution guidelines](#contribution-guidelines) to start working on the issue
 
 Remember to feel free to ask for help by leaving a comment within the Issue.
 
-Working on your first Pull Request? You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+Working on your first pull request? You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
 
 ###### If you've found a bug that is not on the board, [follow these steps](#found-a-bug).
 
 --------------------------------------------------------------------------------
 
-## Contribution Guidelines
+## Contribution guidelines
 
 - [Prerequisites](#prerequisites)
-- [Forking The Project](#forking-the-project)
-- [Create A Branch](#create-a-branch)
-- [Setup for cmd2 development](#setup-for-cmd2-development)
-- [Make Changes](#make-changes)
-- [Static Code Analysis](#static-code-analysis)
-- [Run The Test Suite](#run-the-test-suite)
-- [Squash Your Commits](#squash-your-commits)
-- [Creating A Pull Request](#creating-a-pull-request)
-- [Common Steps](#common-steps)
-- [How We Review and Merge Pull Requests](#how-we-review-and-merge-pull-requests)
-- [How We Close Stale Issues](#how-we-close-stale-issues)
-- [Next Steps](#next-steps)
+- [Forking the project](#forking-the-project)
+- [Creating a branch](#creating-a-branch)
+- [Setting up for cmd2 development](#setting-up-for-cmd2-development)
+- [Making changes](#making-changes)
+- [Static code analysis](#static-code-analysis)
+- [Running the test suite](#running-the-test-suite)
+- [Squashing your commits](#squashing-your-commits)
+- [Creating a pull request](#creating-a-pull-request)
+- [Common steps](#common-steps)
+- [How we review and merge pull requests](#how-we-review-and-merge-pull-requests)
+- [How we close stale issues](#how-we-close-stale-issues)
+- [Next steps](#next-steps)
 - [Other resources](#other-resources)
 - [Advice](#advice)
 - [Developing in an IDE](#developing-in-an-ide)
```

Comment:
At the end of this table of contents, do we want to add links to "Publishing a new release" and "Acknowledgment"?

Reply:
Yes please.  And also the "Found a bug" section.

Reply:
I added the link to the "Publishing a new release" section, but opted to put the "Found a bug" section at the end of the Readme.md file instead.